auth-web/src/components/auth/MemberDataTable.vue

```diff
@@ -307,17 +308,35 @@ export default class MemberDataTable extends Vue {
   }
 
   private get indexedOrgMembers () {
-    // eslint-disable-next-line no-console
-    var self = this
-    this.activeOrgMembers.forEach(function (element) {
-      element.roleDisplayName = self.roleInfos.find(
-        role => role.name === element.membershipTypeCode
+    let orgMembers = []
+    if (this.userNamefilterText) {
+      // filter if the filter by username prop is available
+      orgMembers = this.activeOrgMembers.filter((element) => {
+        const username = `${element.user?.firstname || ''} ${element.user?.lastname || ''}`.trim()
+        const found = username.match(new RegExp(this.userNamefilterText, 'i'))
```

Comment:
match returns true or false rite? I am not 100% sure
if thats the case , why to check the length?

Reply:
no match returns the array of matching values, so length check is needed here